presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -892,6 +894,22 @@ public HiveClientConfig setAssumeCanonicalPartitionKeys(boolean assumeCanonicalP
         return this;
     }
 
+    @MinDataSize("1B")
+    @MaxDataSize("1GB")
+    @NotNull
+    public DataSize getMaxLineLength()
+    {
+        return maxLineLength;
+    }
+
+    @Config("hive.text.line-truncation-limit")
+    @ConfigDescription("Text file lines larger than this will be silently truncated")
```

Comment:
Do we really want silent truncation? I feel like failing loudly for larger lines is better (in terms of correctness) than silently returning a subset of data. We have other similar limits where we fail loudly, e.g., `SliceDirectStreamReader::readBlock()` we have a 1GB limit.